### DISCUSSION TITLE
Does anyone know what the icon next to linters means?

### DISCUSSION THREAD

**Discussion by alexshliu** [golangci/golangci-lint#2931] [Q&A] [answered]
There is an icon next to the linter name on [linters](https://golangci-lint.run/usage/linters/) page. Does anyone know what it means?

**Comment by bombsimon** [comment, marked as answer]
You mean this? 
![image](https://user-images.githubusercontent.com/3248948/174735625-11aff6a2-4f48-4890-9a58-e4748f73e430.png)

It's a link to quickly jump to the linter's available configuration.

**Reply by alexshliu** [reply]
Thank you very much for the reply. Do you know how to choose the best linter? There are so many linters; Is there a list for the best choice?

**Reply by bombsimon** [reply]
There is no combination that fits everyone so you kinda have to read up on each linter.

If you don't know where to start, I would go with the default preset (and not create any custom configuration at all).

**Reply by alexshliu** [reply]
Thanks again. I will read the doc about linter carefully.
